LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Ahmed Salah
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
```

Comment:
- [ ] The MIT license link is broken, Please add a valid license link to make your README file look more professional.

![image](https://github.com/ahmed025566/hello-react-front-end/assets/68796232/fd1744d0-6831-4663-8cab-4fbb156543ed)

Reply:
- You have done well so far, but according to the previous reviewer, the MIT license link in the README file is broken, you are using [this link](https://github.com/ahmed025566/hello-react-front-end/blob/feature/MIT.md) instead of [this](https://github.com/ahmed025566/hello-react-front-end/blob/feature/LICENSE). Please fix this 🙏